Cargo.toml

```diff
@@ -117,14 +117,13 @@ vlog = "0.1.4"
 twoway = "0.2.0"
 cop_datatype = { path = "components/cop_datatype" }
 panic_hook = { path = "components/panic_hook" }
-tipb = { git = "https://github.com/pingcap/tipb.git" }
-kvproto = { git = "https://github.com/pingcap/kvproto.git" }
+tipb = { git = "https://github.com/pingcap/tipb.git", rev = "833c2ffd" }
```

Comment:
You can use the `[patch]` section for `tipb`, `kvproto` and `rocksdb`. Otherwise many files need to be updated to update a single dep.

Reply:
Because of https://github.com/rust-lang/cargo/issues/5478 it's currently not possible.

Reply:
Then I think we should wait it to be resolved, since this PR requires us to update so many fragmentary `Cargo.toml` frequently.

Reply:
Since it seems to be long term waiting, I'm going to close this PR.